benchmarks/checkout.py

```diff
@@ -0,0 +1,25 @@
+import shutil
+
+from benchmarks.base import BaseBench
+
+
+class CheckoutBench(BaseBench):
+    repeat = 3
+
+    params = ["copy", "symlink", "hardlink"]
+    param_names = ["link_type"]
+
+    def setup(self, link_type):
+        super().setup()
+
+        self.init_git()
+        self.init_dvc()
+
+        self.dvc("config", "cache.type", link_type, "--quiet")
+
+        self.gen("data", "cats_dogs")
```

Comment:
I would use an explicit `template` just to avoid the confusion. E.g. how in dvc we have `tmp_dir.gen("foo", "content")`. Just a bit more robust.